applications/nrf5340_audio/unicast_client/README.rst

```diff
@@ -0,0 +1,188 @@
+Transmitting unidirectional or transceiving bidirectional audio happens using CIG and CIS.
+
+The following limitations apply to this application:
+
+* One CIG with two CIS.
+* Audio input: USB or I2S (Line in or using Pulse Density Modulation).
+* Audio output: USB or I2S/Analog headset output.
+* Configuration: 16 bit, several bit rates ranging from 32 kbps to 124 kbps.
+
+.. _nrf53_audio_unicast_client_app_requirements:
+
+Requirements
+************
+
+The application shares the :ref:`requirements common to all nRF5340 Audio application <nrf53_audio_app_requirements>`.
+
+.. _nrf53_audio_unicast_client_app_ui:
+
+User interface
+**************
+
+This application uses specific mapping for the following user interface elements:
+
+* **VOL-** - Pressed on the headset or the CIS gateway during playback: Turns the playback volume down (and unmutes).
+* **VOL+** - Pressed on the headset or the CIS gateway during playback: Turns the playback volume up (and unmutes).
+* **BTN 4** - Pressed on the gateway during playback: Sends a test tone generated on the device. Use this tone to check the synchronization of headsets.
+* **LED1** - Blinking blue on the CIS gateway: Kit has connected to a headset.
+* **RGB1** - Solid green - The device is programmed as the gateway.
+
+All other mappings are common across all nRF5340 Audio applications.
+See the :ref:`nrf53_audio_app_ui` page for detailed overview.
+
+.. _nrf53_audio_unicast_client_app_configuration:
+
+Configuration
+*************
+
+See :ref:`nrf53_audio_app_configuration` and :ref:`nrf53_audio_app_fota` for configuration options common to all nRF5340 Audio applications.
+
+.. _nrf53_audio_unicast_client_app_building:
+
+Building and running
+********************
+
+This application can be found under :file:`applications/nrf5340_audio/unicast_client` in the nRF Connect SDK folder structure.
+
+The nRF5340 Audio DK comes preprogrammed with basic firmware that indicates if the kit is functional.
+See :ref:`nrf53_audio_app_dk_testing_out_of_the_box` for more information.
+
+To build the application, follow the |NCS|'s :ref:`standard methods for building and programming applications <programming>` and for :ref:`modifying them <modifying>`.
+
+You can use the following application-specific parameters to customize the building process:
+
++-------------------+---------------------------------+-----------------------------------------------------------------+
+| Coommand          | Parameter                       | Description                                                     |
++===================+=================================+=================================================================+
+| ``west build``    | ``-DSHIELD=nrf21540ek_fwd``     | Adds :ref:`FEM support <nrf53_audio_app_adding_FEM_support>`.   |
++-------------------+---------------------------------+-----------------------------------------------------------------+
+
+.. _nrf53_audio_unicast_client_app_testing:
+
+Testing
```

Comment:
I'm wondering if this should be somewhere else, so that we can have one 'Testing' for CIS and one for BIS.